### PR TITLE
Add notes about Xcode12

### PR DESCRIPTION
**Pull request by vixentael**
Add notes about which Themis versions to use for Xcode 11 and Xcode 12. Explain ARM64 slice issue.

### CODE REVIEW

File: themis/languages/objc/installation.md [hunk 1]
```diff
@@ -14,6 +14,19 @@ Please [drop us an email](mailto:dev@cossacklabs.com) if you need assistance.
 
 ## Installing with CocoaPods
 
+{{< hint info >}}
+Xcode 12 has introduced ARM64 macos architecture, which Themis currently doesn't support. Follow the issue [#725](https://github.com/cossacklabs/themis/issues/725) for updates.
```
Comment (ilammy) [inline themis/languages/objc/installation.md:18]: Capitalization, clarity:
```suggestion
Xcode 12 has introduced Apple Silicon support on macOS. Themis currently doesn't support the new ARM64 architecture on macOS. Follow the issue [#725](https://github.com/cossacklabs/themis/issues/725) for updates.
```

File: themis/languages/objc/installation.md [hunk 2]
```diff
@@ -14,6 +14,19 @@ Please [drop us an email](mailto:dev@cossacklabs.com) if you need assistance.
...
+
+**iOS:**
+Xcode 12: use `pod themis, "0.13.3"`. It excludes arm64-iphonesimulator architecture slice for now. Resulting app can be submitted to App Store.
+Xcode 11: use `pod themis, "0.13.0"`. It is compatible with Xcode 11, not with Xcode 12.
+
+See [#715](https://github.com/cossacklabs/themis/issues/715), [#725](https://github.com/cossacklabs/themis/issues/725).
```
Comment (ilammy) [inline themis/languages/objc/installation.md:24]: IIRC, Hugo interprets line breaks as traditional Markdown/HTML (i.e., single line break = space). I think lists will be appropriate here. Plus, i think a bit more explanations won't hurt.

```suggestion
**iOS apps:**

  - Xcode 12: use `pod themis, "0.13.3"`. It excludes arm64-iphonesimulator architecture slice for now. Resulting app can be submitted to App Store.
  - Xcode 11: use `pod themis, "0.13.0"` for now. Newer releases are not compatible with older Xcode version. We are working on improved compatbility.

See issues [#715](https://github.com/cossacklabs/themis/issues/715), [#725](https://github.com/cossacklabs/themis/issues/725).
```

File: themis/languages/objc/installation.md [hunk 3]
```diff
@@ -45,11 +58,24 @@ pod 'themis/themis-boringssl'
...
 Unfortunately, BoringSSL for iOS doesn’t support _bitcode_,
 so Themis with BoringSSL also doesn’t support bitcode :(
-This might prevent your app from being accepted to the App Store.
+Please disable bitcode for your application before archiving it to App Store, and everything will be fine.
 {{< /hint >}}
 
 ## Installing with Carthage
 
+{{< hint info >}}
+Xcode 12 has introduced ARM64 macos architecture, which Themis currently doesn't support. Follow the issue [#725](https://github.com/cossacklabs/themis/issues/725) for updates and exclude ARM64 architecture for now https://github.com/cossacklabs/themis/issues/713.
```
Comment (ilammy) [inline themis/languages/objc/installation.md:67]: Capitalization, clarity:
```suggestion
Xcode 12 has introduced Apple Silicon support on macOS. Themis currently doesn't support the new ARM64 architecture on macOS. Follow the issue [#725](https://github.com/cossacklabs/themis/issues/725) for updates and exclude ARM64 architecture for now, see issue [#713](https://github.com/cossacklabs/themis/issues/713) for instructions.
```

File: themis/languages/objc/installation.md [hunk 4]
```diff
@@ -45,11 +58,24 @@ pod 'themis/themis-boringssl'
...
+
+**iOS:**
+Exclude arm64-iphonesimulator architecture slice for now (see https://github.com/cossacklabs/themis/issues/713).
```
Comment (ilammy) [inline themis/languages/objc/installation.md:70]: Links:
```suggestion
Exclude arm64-iphonesimulator architecture slice for now (see issue [#713](https://github.com/cossacklabs/themis/issues/713)).
```

File: themis/languages/objc/installation.md [hunk 5]
```diff
@@ -45,11 +58,24 @@ pod 'themis/themis-boringssl'
...
+
+**macOS:**
+If you use macOS, use `0.13.3`, and please exclude mac-arm64 for now (set `Build Active Architecture Only = YES`). This will build correct x86_64 application.
+
+See https://github.com/cossacklabs/themis/issues/725, https://github.com/cossacklabs/themis/issues/713.
```
Comment (ilammy) [inline themis/languages/objc/installation.md:75]: Links:
```suggestion
See issues [#725](https://github.com/cossacklabs/themis/issues/725), [#713](https://github.com/cossacklabs/themis/issues/713).
```

**Comment by vixentael** [comment]
Thank you @ilammy for valid suggestions. All accepted.

**Review by ilammy — APPROVED** [review]
:+1:

**Pull request merged**